src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -156,16 +157,18 @@ public int allowedSpaceCount()
 
     @NotNull
     public List<String> getTabCompletions(@NotNull final MinecraftServer server,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 7, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)